src/brokers/broker-details/components/Topics/Topics.component.tsx

```diff
@@ -28,24 +28,26 @@ const Topics: React.FC<TopicsProps> = ({ topicData, isLoaded, loadError }) => {
   ];
   return (
     <>
-      <PageSection>
+      <div className="pf-u-mt-md pf-u-ml-md pf-u-mb-sm">
         <Title headingLevel="h1">{t('topics')}</Title>
-      </PageSection>
-      <VirtualizedTable<Topic>
-        data={topicData}
-        unfilteredData={topicData}
-        loaded={isLoaded}
-        loadError={loadError}
-        columns={columns}
-        Row={({ obj, activeColumnIDs, rowData }) => (
-          <TopicsRow
-            obj={obj}
-            rowData={rowData}
-            activeColumnIDs={activeColumnIDs}
-            columns={columns}
-          />
-        )}
-      />
+      </div>
+      <div>
```

Comment:
Do we need this extra `div` ?

Reply:
> Do we need this extra div ? 
->  not necessary i was just checking by adding this does it stop crashing table alignment issue. i will remove it.